README.md

```diff
@@ -44,6 +44,18 @@ apply plugin: 'com.apollographql.android'
 
 The Android Plugin must be applied before the Apollo plugin
 
+### Kotlin
+
+If using Apollo in your Kotlin project, make sure to apply the Apollo plugin before your Kotlin plugins within your app module's `build.gradle`:
+
+```
+apply plugin: 'com.apollographql.android'
+// Kotlin plugins
```

Comment:
I just wanted to put it there to emphasize the order of the plugins. Can't hurt to be too explicit right?